extensions/clusters/gke/gke_cluster_config.go

```diff
@@ -143,10 +143,14 @@ func clusterAddonsBuilder(clusterAddons *ClusterAddons) *management.GKEClusterAd
 }
 
 func cmekConfigBuilder(cmekConfig *CMEKConfig) *management.CMEKConfig {
-	return &management.CMEKConfig{
-		KeyName:  cmekConfig.KeyName,
-		RingName: cmekConfig.RingName,
+	var CMEKConfig *management.CMEKConfig
+	if cmekConfig != nil {
```

Comment:
This needs the same fix Chandan as the main branch

Reply:
Done, thanks.